typosquat.rego

```diff
@@ -0,0 +1,30 @@
+package policy
+
+import data.phylum.ecosystem
+import data.phylum.domain
+import data.phylum.level
+
+import future.keywords.contains
+import future.keywords.if
+
+# Returns `true` if the given dependency has a typosquat issue
+has_typosquat {
+    issues := data.dependency.issues
+    count([tag | issues[i].tag == "HM0008"; tag := issues[i].tag]) == 1

```

Comment:
There is no need for `count` here. Unless a dependency with 2 typosquat issues is somehow better than only having 1.

```suggestion
    some i
    data.dependency.issues[i].tag == "HM0008"
```

Reply:
Not even sure that is possible come to think of it 🤔